Density/app/src/main/res/values/styles.xml

```diff
@@ -36,10 +36,6 @@
         <item name="queryBackground">@android:color/transparent</item>
     </style>
 
-    <style name="ChipTextStyle">
-        <item name="android:textSize">13sp</item>
-    </style>
-
```

Comment:
Why is this removed and the other xml uses the hardcoded value `13sp`?

Reply:
I think I might add it back as a value instead of a style because having a whole style just to set text size seems unnecessary.